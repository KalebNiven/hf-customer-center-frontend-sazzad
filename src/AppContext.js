import React, { createContext, useContext, useState, useEffect } from "react";

export const AppContext = createContext();

export const AppContextProvider = ({ children }) => {
  const [drawerOpen, setDrawerOpen] = useState(false);
  const [externalSiteModal, setExternalSiteModal] = useState({
    isVisible: false,
    link: "#",
    target: "_blank",
    segmentPageCategory: null,
    segmentTitle: null,
    segmentTargetMemberId: null,
    membershipKey: null,
    label: null,
  });
  const [innerWidth, setInnerWidth] = useState("");
  const [toastOpen, setToastOpen] = useState(false);
  const [toastState, setToastState] = useState("");
  const [planName, setPlanName] = useState("");
  const [openPaperLess, setOpenPaperLess] = useState(false);
  const [globalError, setGlobalError] = useState(false);
  const [acknowledgmentModal, setAcknowledgmentModal] = useState({
    isVisible: false,
  });

  const resetExternalModal = () => {
    setExternalSiteModal({
      isVisible: false,
      link: null,
      target: null,
      pageCategory: null,
      segmentPageCategory: null,
      segmentTitle: null,
      segmentTargetMemberId: null,
      membershipKey: null,
      label: null,
    });
  };

  const resetAcknowledgmentModal = () => {
    setAcknowledgmentModal({ isVisible: false });
  };
  /* Window Resize Listener */
  const onWindowResize = () => {
    const innerWidth = window.innerWidth;
    setInnerWidth(innerWidth);
  };

  useEffect(() => {
    window.addEventListener("resize", onWindowResize);
    const innerWidth = window.innerWidth;
    setInnerWidth(innerWidth);

    return () => {
      window.removeEventListener("resize", onWindowResize);
    };
  }, []);

  return (
    <AppContext.Provider
      value={{
        externalSiteModal,
        setExternalSiteModal,
        openPaperLess,
        setOpenPaperLess,
        toastState,
        setToastState,
        planName,
        setPlanName,
        toastOpen,
        setToastOpen,
        drawerOpen,
        setDrawerOpen,
        externalSiteModal,
        setExternalSiteModal,
        resetExternalModal,
        innerWidth,
        setInnerWidth,
        globalError,
        setGlobalError,
        acknowledgmentModal,
        setAcknowledgmentModal,
        resetAcknowledgmentModal,
      }}
    >
      {children}
    </AppContext.Provider>
  );
};

export const useAppContext = () => {
  return useContext(AppContext);
};
