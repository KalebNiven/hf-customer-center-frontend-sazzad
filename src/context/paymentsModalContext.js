import React, { createContext, useContext, useState } from "react";

export const PaymentsModalContext = createContext();

export const PaymentsModalContextProvider = ({ children }) => {
  const initState = {
    routeLink: "",
    externalLinkName: "",
    showMemberModal: false,
    membershipKey: null,
    membership: null,
  };
  const [paymentsModalState, setPaymentsModalState] = useState({
    ...initState,
  });

  const resetPaymentsModal = () => {
    setPaymentsModalState({ ...initState });
  };

  return (
    <PaymentsModalContext.Provider
      value={{ paymentsModalState, setPaymentsModalState, resetPaymentsModal }}
    >
      {children}
    </PaymentsModalContext.Provider>
  );
};

export const usePaymentsModalContext = () => {
  return useContext(PaymentsModalContext);
};
