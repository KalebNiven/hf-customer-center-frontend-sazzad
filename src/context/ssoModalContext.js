import React, { createContext, useContext, useState } from "react";

export const SSOModalContext = createContext();

export const SSOModalContextProvider = ({ children }) => {
  const initState = {
    routeLink: "",
    externalLinkName: "",
    showMemberModal: false,
    membershipKey: null,
  };
  const [ssoModalState, setSsoModalState] = useState({ ...initState });

  const resetSsoModal = () => {
    setSsoModalState({ ...initState });
  };

  return (
    <SSOModalContext.Provider
      value={{ ssoModalState, setSsoModalState, resetSsoModal }}
    >
      {children}
    </SSOModalContext.Provider>
  );
};

export const useSSOModalContext = () => {
  return useContext(SSOModalContext);
};
