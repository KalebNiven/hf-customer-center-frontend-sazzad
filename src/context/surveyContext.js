import React, { createContext, useContext, useState } from "react";
export const SurveyContext = createContext();
const HRA_WIDGET_SCRIPT_ID = "survey-widget-script";
const DIGITAL_SURVEY_EVENTS = {
  CLAIM_SUBMITTED: "CLAIM_SUBMITTED",
  MAIL_ID_CARD: "MAIL_ID_CARD",
  PCP_UPDATE: "PCP_UPDATE",
  CHAT: "CC_CHAT",
};

export const SurveyContextProvider = ({ children }) => {
  const [surveyScript, setSurveyScript] = useState(
    document.getElementById(HRA_WIDGET_SCRIPT_ID)
  );
  const [digitalSurveyWidget, setDigitalSurveyWidget] = useState(null);

  const resetDigitalSurveyWidget = () => {
    if (surveyScript && digitalSurveyWidget) {
      digitalSurveyWidget.unmount("all");
      setDigitalSurveyWidget(null);
    }
  };

  const triggerDigitalSurveyByEventName = (widget, eventName) => {
    if (!widget) new Error("Survey Widget not found");
    if (!eventName) new Error("Event name not provided");
    widget.action = eventName;
    return true;
  };

  return (
    <SurveyContext.Provider
      value={{
        surveyScript,
        setSurveyScript,
        digitalSurveyWidget,
        setDigitalSurveyWidget,
        DIGITAL_SURVEY_EVENTS,
        triggerDigitalSurveyByEventName,
        resetDigitalSurveyWidget,
      }}
    >
      {children}
    </SurveyContext.Provider>
  );
};

export const useSurveyContext = () => {
  return useContext(SurveyContext);
};
