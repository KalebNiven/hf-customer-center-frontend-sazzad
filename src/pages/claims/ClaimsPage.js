import { useClient } from "@splitsoftware/splitio-react";
import React from "react";
import PermissionDenied from "../../components/common/PermissionDenied";
import ClaimsPage from "../../components/claims/claimsPage";
import ClaimsWidget from "../../components/claims/ClaimsWidget";
import { CLAIMS_PAGE } from "../../constants/splits";
import PageLayout from "../../layouts/PageLayout";

export default () => {
  const splitHookClient = useClient();
  const claimsPageSplitTreatment = splitHookClient.getTreatment(CLAIMS_PAGE);

  switch (claimsPageSplitTreatment) {
    case "on":
      return <ClaimsWidget />;
    case "legacy":
      return <ClaimsPage />;
    case "off":
      return <PermissionDenied />;
    case "control":
      return <></>;
    default:
      return <></>;
  }
};
