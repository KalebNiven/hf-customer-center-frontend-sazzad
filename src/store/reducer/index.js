import { combineReducers } from "redux";
import claim from "./claimReducer";
import claimDetails from "./claimDetailsReducer";
import authorization from "./authorizationReducer";
import authorizationDetails from "./authorizationDetailsReducer";
import oop from "./oopReducer";
import coverageBenefits from "./coverageBenefitsReducer";
import customerInfo from "./customerInfoReducer";
import customerDemographicsInfo from "./customerDemographicsInfoReducer";
import pcpDetails from "./pcpDetailsReducer";
import pcpStatus from "./pcpStatusReducer";
import hra from "./hraReducer";
import physicalIdCard from "./physicalIdCardReducer";
import digitalIdCard from "./digitalIdCardReducer";
import correspondence from "./correspondenceReducer";
import correspondenceStatus from "./correspondenceStatusReducer";
import selectedMember from "./selectedMemberReducer";
import settingsReducer from "./settingsReducer";
import myHealth from "./myHealthReducer";
import homeDetails from "./homeReducer";
import addMembership from "./addMembershipReducer";
import submitClaimDetails from "./submitClaimDetailsReducer";
import submitContactInfoPayload from "./updateContactInfoReducer";
import submitAttestationAgreement from "./submitAttestationAgreementReducer";
import preferenceCenterInfo from "./preferenceCenterInfoReducer";
import preferredContactInfoSubmit from "./preferredContactInfoSubmitReducer";
import mfaCode from "./mfaCodeReducer";
import mfaVerify from "./mfaVerifyReducer";
import globalAlerts from "./globalAlertsReducer";
import otcCard from "./OTCCardReducer";
import mfaFactors from "./mfaFactors";
import memberRegister from "./memberRegister";
import createUsernamePassword from "./createUsernamePasswordReducer";
import * as actionTypes from "../actions/actionTypes";
import forgotUsername from "./forgotUsernameReducer";
import forgotPassword from "./forgotPasswordReducer";
import setPassword from "./setPasswordReducer";
import documents from "./documentsReducer";
import userMfaCode from "./userMfaCodeReducer";
import userMfaVerify from "./userMfaVerifyReducer";
import pcp from "./pcpReducer";
import selectPlan from "./selectPlanReducer";
import verifyAddress from "./verifyAddressReducer";
import pcpHousehold from './pcpHousehold';
import ccFormsDoc from "./ccFormsDocReducer";

const appReducer = combineReducers({
  claim,
  claimDetails,
  authorization,
  authorizationDetails,
  oop,
  coverageBenefits,
  customerInfo,
  customerDemographicsInfo,
  pcpStatus,
  pcpDetails,
  hra,
  physicalIdCard,
  digitalIdCard,
  settingsReducer,
  correspondence,
  selectedMember,
  myHealth,
  homeDetails,
  addMembership,
  submitAttestationAgreement,
  submitClaimDetails, 
  correspondenceStatus,
  submitContactInfoPayload,
  preferenceCenterInfo,
  preferredContactInfoSubmit,
  mfaCode,
  mfaVerify,
  mfaFactors,
  globalAlerts,
  otcCard,
  memberRegister,
  createUsernamePassword,
  forgotUsername,
  forgotPassword,
  setPassword,
  documents,
  userMfaCode,
  userMfaVerify,
  pcp,
  selectPlan,
  verifyAddress,
  pcpHousehold,
  ccFormsDoc
});

export default (state, action) => {
  if (action.type === actionTypes.REQUEST_RESET_STATE) {
    return appReducer(undefined, action)
  }

  return appReducer(state, action)
}