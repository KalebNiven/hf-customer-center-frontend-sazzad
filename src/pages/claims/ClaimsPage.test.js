import React from "react";
import ClaimsPage from "../../components/claims/claimsPage";

import { render } from "@testing-library/react";
import { Provider } from "react-redux";

import { devTools } from "../../store/store";
import { SurveyContextProvider } from "../../context/surveyContext";
import { AppContextProvider } from "../../AppContext";
import createMockReduxStore from "../../test-resources/createMockReduxStore";
import 'jest-styled-components'

describe("ClaimsPage", () => {
  let store;

  beforeEach(() => {
    store = createMockReduxStore();
  });

  it("renders correctly", () => {
    const { asFragment } = render(
      <AppContextProvider>
        <SurveyContextProvider>
          <Provider store={store}>
            <ClaimsPage ignoreSplit={true} />
          </Provider>
        </SurveyContextProvider>
      </AppContextProvider>
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
