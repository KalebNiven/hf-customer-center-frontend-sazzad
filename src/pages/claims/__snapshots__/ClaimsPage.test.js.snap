// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ClaimsPage renders correctly 1`] = `
<DocumentFragment>
  .c11 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex: auto;
  -ms-flex: auto;
  flex: auto;
  width: 25%;
  margin-right: 10px;
  color: #474b55;
}

.c11 * {
  box-sizing: border-box !important;
}

.c12 {
  border-radius: 4px;
  border: solid 1px #a8abac;
  background-color: #ffffff;
  cursor: pointer;
  text-align: left;
  margin: auto;
  width: 100%;
  position: relative;
  margin-bottom: 8px;
  width: -webkit-fill-available;
  padding: 9px 0px 9px 0px;
  background-color: #ffffff;
  font-size: 16px;
  font-weight: 300;
  font-family: "museo-sans";
  background: url(react/images/icons-solid-user.svg) no-repeat scroll #ffffff 10px 13px;
}

.c12:focus {
  box-shadow: none;
}

.c12:focus .hf-preference-center-dropdown-content {
  display: block;
}

.c12.selected {
  background: url(react/images/icons-solid-user-dark-grey.svg) no-repeat scroll #ffffff 10px 13px;
}

.c14 {
  font-size: 16px;
  font-weight: 300;
  font-stretch: normal;
  font-style: normal;
  -webkit-letter-spacing: normal;
  -moz-letter-spacing: normal;
  -ms-letter-spacing: normal;
  letter-spacing: normal;
  color: #a8abac;
  padding: 0px 0px 0px 0px;
  display: inline-block;
  vertical-align: middle;
  white-space: nowrap;
  overflow: hidden;
  text-overflow: ellipsis;
  line-height: 24px;
  width: calc(100% - 40px);
}

.c14.selected {
  color: #474b55;
}

.c14.error {
  color: #ad122a;
}

.c13 {
  width: 100%;
  padding: 0px 0px 0px 35px;
}

.c15 {
  display: inline-block;
}

.c16 {
  position: relative;
  width: 20px;
  height: 20px;
  object-fit: cover;
  overflow: hidden;
  margin: -5px 8px -4px 8px;
  vertical-align: middle;
}

.c16.selected {
  color: #474b55;
}

.c17 {
  display: none;
  position: absolute;
  background-color: #ffffff;
  max-height: 298px;
  margin: 13px 0px;
  border-radius: 4px;
  box-shadow: 0 2px 8px 0 #d8d8d8;
  background-color: #ffffff;
  list-style-type: none;
  padding: 6px 0px;
  overflow: hidden;
  overflow-y: auto;
  z-index: 1;
  width: 100%;
}

.c17::-webkit-scrollbar {
  width: 8px;
}

.c17::-webkit-scrollbar-track {
  background: #f1f1f1;
  border-radius: 4px;
}

.c17::-webkit-scrollbar-thumb {
  background: #d8d8d8;
  border-radius: 4px;
}

.c17::-webkit-scrollbar-thumb:hover {
  background: #c3c3ce;
}

.c18 {
  font-size: 16px;
  font-weight: 300;
  font-stretch: normal;
  font-style: normal;
  line-height: 1.5;
  -webkit-letter-spacing: normal;
  -moz-letter-spacing: normal;
  -ms-letter-spacing: normal;
  letter-spacing: normal;
  color: #474b55;
  text-align: left;
  border: none;
  background-color: #ffffff;
  padding: 9px 12px;
  display: block;
  cursor: pointer;
  z-index: 2;
  width: inherit;
}

.c18:focus {
  box-shadow: none;
}

.c18:hover {
  background-color: #f8f8f8;
}

.c18 h1 {
  font-weight: 500;
  font-family: "museo-sans";
  font-size: 16px;
  color: #474b55;
  padding: 0;
  margin: 0;
}

.c18 h2 {
  font-weight: 300;
  font-family: "museo-sans";
  font-size: 16px;
  color: #474b55;
  padding: 0;
  margin: 0;
}

.c18 p {
  font-weight: 300;
  font-family: "museo-sans";
  font-size: 12px;
  color: #474b55;
  padding: 0;
  margin: 0;
}

.c20 {
  padding: 8px 0px 8px 0px;
  margin-bottom: 0px;
}

.c22 {
  padding: 8px 0px 8px 0px;
}

.c9 {
  display: contents;
  -webkit-flex: 1 1 auto;
  -ms-flex: 1 1 auto;
  flex: 1 1 auto;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  box-sizing: border-box;
  width: 100%;
  min-width: 600px;
}

.c8 {
  margin: 10px 16px 12px;
  margin-bottom: 20rem;
}

.c10 {
  height: 27px;
  border-radius: 4px;
  border: solid 1px #a8abac;
  -webkit-align-self: flex-end;
  -ms-flex-item-align: end;
  align-self: flex-end;
  margin-bottom: 16px;
  box-sizing: initial;
  width: -webkit-fill-available;
  padding: 8px 0px 8px 35px;
  background-color: #ffffff;
  font-size: 16px;
  font-weight: 300;
  font-family: "museo-sans";
  background: url("/react/images/icn-search.svg") no-repeat scroll #ffffff 10px 13px;
}

.c10::-webkit-input-placeholder {
  color: #a8abac;
}

.c10::-moz-placeholder {
  color: #a8abac;
}

.c10:-ms-input-placeholder {
  color: #a8abac;
}

.c10:-moz-placeholder {
  color: #a8abac;
}

.c10::-webkit-input-placeholder {
  color: #a8abac;
}

.c10::-moz-placeholder {
  color: #a8abac;
}

.c10:-ms-input-placeholder {
  color: #a8abac;
}

.c10::placeholder {
  color: #a8abac;
}

.c10:focus {
  border: solid 2px #003863;
}

.c19 {
  margin-bottom: 20px;
  margin-top: -7px;
}

.c21 {
  cursor: pointer;
}

.c0 {
  height: 100%;
}

.c24 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  width: 40px;
  height: 64px;
  background-color: #36502a;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c25 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  width: 16px;
  height: 16px;
}

.c26 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  width: 288px;
  height: 64px;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: space-around;
  -webkit-justify-content: space-around;
  -ms-flex-pack: space-around;
  justify-content: space-around;
}

.c28 {
  width: 16px;
  height: 16px;
  object-fit: contain;
  cursor: pointer;
}

.c27 {
  font-size: 14px;
  font-weight: 500;
  line-height: 1.29;
  color: white;
}

.c23 {
  position: fixed;
  background-color: #3e7128;
  top: 190px;
  left: 50%;
  -webkit-transform: translate(-50%,-50%);
  -ms-transform: translate(-50%,-50%);
  transform: translate(-50%,-50%);
  background-color: #3e7128;
  display: none;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
}

.c1 {
  background-color: #f4f4f4;
  max-width: 1024px;
  position: relative;
  -webkit-align-self: center;
  -ms-flex-item-align: center;
  align-self: center;
  width: 100%;
  margin-bottom: 1rem;
}

.c7 {
  height: 1px;
  background-color: #d8d8d8;
  margin: 0px 16px 0px;
  min-width: 600px;
}

.c2 {
  font-size: 24px;
  font-weight: bold;
  font-stretch: normal;
  font-style: normal;
  line-height: 1.33;
  -webkit-letter-spacing: normal;
  -moz-letter-spacing: normal;
  -ms-letter-spacing: normal;
  letter-spacing: normal;
  color: #003863;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-self: flex-end;
  -ms-flex-item-align: end;
  align-self: flex-end;
}

.c5 {
  visibility: hidden;
}

.c3 {
  font-size: 16px;
  font-weight: 300;
  font-stretch: normal;
  font-style: normal;
  line-height: 1.5;
  -webkit-letter-spacing: normal;
  -moz-letter-spacing: normal;
  -ms-letter-spacing: normal;
  letter-spacing: normal;
  color: #474b55;
  margin-right: 73px;
}

.c4 {
  font-size: 12px;
  font-weight: 500;
}

.c6 {
  margin: 0px 16px 0px;
}

@media only screen and (max-width:768px) {
  .c11 {
    width: 100%;
  }
}

@media only screen and (max-width:768px) {
  .c17 {
    width: 100%;
  }
}

@media only screen and (min-width:770px) {
  .c20 {
    padding: 8px 0px 8px 0px;
    margin: 0px 10px 0px 0px;
  }
}

@media only screen and (min-width:769px) and (max-width:769px) {
  .c20 {
    margin: 16px 10px 0px 0px;
  }
}

@media only screen and (min-width:770px) {
  .c22 {
    padding: 8px 0px 8px 0px;
    margin-top: 0px;
  }
}

@media only screen and (min-width:769px) and (max-width:769px) {
  .c22 {
    margin: 16px 0px 0px 0px;
  }
}

@media only screen and (min-width:769px) {
  .c9 {
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
  }
}

@media only screen and (min-width:769px) {

}

@media only screen and (max-width:768px) {
  .c10 {
    margin-top: 15px;
  }
}

@media only screen and (min-width:769px) {
  .c10 {
    margin-right: 10px;
    width: 328px;
  }
}

@media only screen and (min-width:769px) {
  .c19 {
    margin-top: -30px;
    margin-bottom: 8px;
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
  }
}

@media only screen and (min-width:769px) {

}

@media only screen and (min-width:769px) {

}

@media only screen and (min-width:769px) {

}

@media only screen and (min-width:769px) {

}

<div
    class="c0 c1"
  >
    <div
      class="MuiBox-root MuiBox-root-1"
    >
      <div
        class="c2"
      >
        Claims
      </div>
    </div>
    <div
      class="MuiBox-root MuiBox-root-2"
    >
      <div
        class="c3"
      >
        Your Healthfirst account lets you access important information about your health plan. Here you can see the status of your claims.
        <br />
        <br />
        <div
          class="c4"
        >
          Keep in mind that due to sensitive health information, you may not able to view all your claims. If you have any questions, please call the Member Services phone number on your Member ID card for assistance.
        </div>
      </div>
      <div
        class="c5"
      />
    </div>
    <div
      class=""
    >
      <div
        class="c6"
      >
        <div
          class="MuiTabs-root"
        >
          <div
            class="MuiTabs-scroller MuiTabs-fixed"
            style="overflow: hidden;"
          >
            <div
              class="MuiTabs-flexContainer"
              role="tablist"
            >
              <button
                aria-selected="true"
                class="MuiButtonBase-root MuiTab-root MuiTab-textColorInherit Mui-selected"
                role="tab"
                style="font-size: 14px; font-weight: bold; font-stretch: normal; font-style: normal; line-height: 1.45; letter-spacing: 1px; text-align: center; color: rgb(62, 113, 40); text-transform: none; font-family: museo-sans; min-width: 80px; padding: 0px 20px;"
                tabindex="0"
                type="button"
              >
                <span
                  class="MuiTab-wrapper"
                >
                  Claims
                </span>
                <span
                  class="MuiTouchRipple-root"
                />
              </button>
            </div>
            <span
              class="PrivateTabIndicator-root-3 PrivateTabIndicator-colorSecondary-5 MuiTabs-indicator"
              style="left: 0px; width: 0px; background: rgb(62, 113, 40);"
            />
          </div>
        </div>
      </div>
      <div
        class="c7"
      />
    </div>
    <div
      class="c8"
    >
      <div
        class="c9"
      >
        <input
          aria-label="Search Input"
          class="c10"
          id="search"
          placeholder="Search claims #'s, providers, etc."
          type="text"
          value=""
        />
        <div
          class="c11"
        >
          <div
            class="c12 false"
          >
            <div
              class="c13"
            >
              <div
                class="c14 selected false"
              >
                All Members
              </div>
              <div
                class="c15"
              >
                <img
                  alt=""
                  class="c16"
                  src="/react/images/icn-dropdown.svg"
                />
              </div>
            </div>
            <div
              class="c17 hf-preference-center-dropdown-content"
            >
              <button
                class="c18"
                name="All Members"
              >
                <h1>
                  All Members
                </h1>
              </button>
              <button
                class="c18"
                name="Sasha Buffin"
                value="420000000696100"
              >
                <div>
                  <h2>
                    Sasha Buffin
                  </h2>
                  <p>
                    Silver leaf
                  </p>
                </div>
              </button>
              <button
                class="c18"
                name="Sasha Buffin"
                value="964311200"
              >
                <div>
                  <h2>
                    Sasha Buffin
                  </h2>
                  <p>
                    Child health plus state pay
                  </p>
                </div>
              </button>
              <button
                class="c18"
                name="Suzy Buffin"
                value="964311202"
              >
                <div>
                  <h2>
                    Suzy Buffin
                  </h2>
                  <p>
                    Child Health Plus
                  </p>
                </div>
              </button>
              <button
                class="c18"
                name="Amel Buffin"
                value="964311201"
              >
                <div>
                  <h2>
                    Amel Buffin
                  </h2>
                  <p>
                    Child Health Plus
                  </p>
                </div>
              </button>
              <button
                class="c18"
                name="Athanasios Buffin"
                value="964311203"
              >
                <div>
                  <h2>
                    Athanasios Buffin
                  </h2>
                  <p>
                    Child Health Plus
                  </p>
                </div>
              </button>
              <button
                class="c18"
                name="Noriko Buffin"
                value="420000000696101"
              >
                <div>
                  <h2>
                    Noriko Buffin
                  </h2>
                  <p>
                    Silver Leaf
                  </p>
                </div>
              </button>
            </div>
          </div>
        </div>
        <div
          class="c19"
        >
          <div
            class="c20"
          >
            <div
              class="MuiFormControl-root MuiTextField-root"
              format="MM/dd/yyyy"
            >
              <div
                class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl MuiInputBase-adornedStart MuiOutlinedInput-adornedStart MuiInputBase-adornedEnd MuiOutlinedInput-adornedEnd"
                style="height: 45px; border: 1px solid #a8abac; font-family: museo-sans; background-color: rgb(255, 255, 255); width: 100%; border-radius: 4px;"
              >
                <img
                  alt=""
                  src="/react/images/icn-calendar.svg"
                />
                <input
                  aria-invalid="false"
                  class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputAdornedStart MuiOutlinedInput-inputAdornedStart MuiInputBase-inputAdornedEnd MuiOutlinedInput-inputAdornedEnd"
                  id="date-start-inline"
                  placeholder="Start Date"
                  readonly=""
                  type="text"
                  value=""
                />
                <img
                  alt=""
                  class="c21"
                  src="/react/images/icn-dropdown.svg"
                />
                <fieldset
                  aria-hidden="true"
                  class="PrivateNotchedOutline-root-7 MuiOutlinedInput-notchedOutline"
                  style="padding-left: 8px;"
                >
                  <legend
                    class="PrivateNotchedOutline-legend-8"
                    style="width: 0.01px;"
                  >
                    <span>
                      ​
                    </span>
                  </legend>
                </fieldset>
              </div>
            </div>
          </div>
          <div
            class="c22"
          >
            <div
              class="MuiFormControl-root MuiTextField-root"
              format="MM/dd/yyyy"
            >
              <div
                class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-formControl MuiInputBase-adornedStart MuiOutlinedInput-adornedStart MuiInputBase-adornedEnd MuiOutlinedInput-adornedEnd"
                style="height: 45px; border: 1px solid #a8abac; font-family: museo-sans; background-color: rgb(255, 255, 255); width: 100%; border-radius: 4px;"
              >
                <img
                  alt=""
                  src="/react/images/icn-calendar.svg"
                />
                <input
                  aria-invalid="false"
                  class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputAdornedStart MuiOutlinedInput-inputAdornedStart MuiInputBase-inputAdornedEnd MuiOutlinedInput-inputAdornedEnd"
                  id="date-end-inline"
                  placeholder="End Date"
                  readonly=""
                  type="text"
                  value=""
                />
                <img
                  alt=""
                  class="c21"
                  src="/react/images/icn-dropdown.svg"
                />
                <fieldset
                  aria-hidden="true"
                  class="PrivateNotchedOutline-root-7 MuiOutlinedInput-notchedOutline"
                  style="padding-left: 8px;"
                >
                  <legend
                    class="PrivateNotchedOutline-legend-8"
                    style="width: 0.01px;"
                  >
                    <span>
                      ​
                    </span>
                  </legend>
                </fieldset>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div>
      <div
        class="c23"
      >
        <div
          class="c24"
        >
          <img
            alt=""
            class="c25"
            src="/react/images/valid-check.svg"
          />
        </div>
        <div
          class="c26"
        >
          <div
            class="c27"
          >
            Claim submitted successfully!
          </div>
          <img
            alt=""
            class="c28"
            src="/react/images/valid-close.svg"
          />
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
